packages/react-server-dom-relay/src/__tests__/ReactFlightDOMRelay-test.internal.js

```diff
@@ -21,8 +21,8 @@ describe('ReactFlightDOMRelay', () => {
     act = require('react-dom/test-utils').unstable_concurrentAct;
     React = require('react');
     ReactDOM = require('react-dom');
-    ReactDOMFlightRelayServer = require('react-transport-dom-relay/server');
-    ReactDOMFlightRelayClient = require('react-transport-dom-relay');
+    ReactDOMFlightRelayServer = require('react-server-dom-relay/server');
+    ReactDOMFlightRelayClient = require('react-server-dom-relay');
```

Comment:
Should we start getting rid of the codename in source? It’s already being referred to in some places which risks confusion.

Reply:
These are internal FB names. I think I’ve changed the external ones.

Reply:
Yeah but I mean people read the source too. If we don't need the codename we could just remove it everywhere.